chapters/unitexpressions.tex

```diff
@@ -62,17 +62,20 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 \end{lstlisting}
 
 A \lstinline[language=grammar]!unit-symbol! is a non-empty string of \lstinline[language=grammar]!unit-char! characters.
-The set of allowed characters is currently not defined.
+The set of allowed characters is currently not defined, but it is recommended to only use \lstinline[language=grammar]!NON-DIGIT!.
 It is required that basic and derived units of the SI system are recognized, but tools are allowed to additionally support user-defined unit symbols.
 The required unit symbols do not make use of Greek letters, but a unit such as $\Omega$ is spelled out as \lstinline!"Ohm"!.
+Similarly degree is spelled out as \lstinline!"deg"!, both on its own (for angles) and as part of \lstinline!"degC"!, \lstinline!"degF"! and \lstinline!"degRk"! for temperatures (Celsius, Fahrenheit and Rankine).
 
 A \lstinline[language=grammar]!unit-operand! should first be interpreted as a \lstinline[language=grammar]!unit-symbol! and only if not successful the second alternative assuming a prefixed operand should be exploited.
 There must be no spacing between the \lstinline[language=grammar]!unit-symbol! and a possible \lstinline[language=grammar]!unit-prefix!.
 The values of the prefixes are according to the ISO standard.
 The letter \lstinline!u! is used as a symbol for the prefix \emph{micro}.
 
 \begin{nonnormative}
-A tool may present \lstinline!Ohm! as $\Omega$ and the prefix \lstinline!u! as $\mu$, and similarly \lstinline!m2! as m\textsuperscript{2}.
+A tool may present \lstinline!"Ohm"! as $\Omega$ and the prefix \lstinline!"u"! as $\mu$, and similarly \lstinline!"m2"! as m\textsuperscript{2}.
+In addition a tool may present \lstinline!"deg"! as $^{\circ}$, both on its own (for angles) and for temperatures - e.g., \lstinline!"degC"! can be written as $^{\circ}$C.
```

Comment:
En-dash, say _presented_ for consistency, and match style of the added content with style of pre-existing rules:
```suggestion
A tool may present \lstinline!"Ohm"! as $\Omega$ and the prefix \lstinline!"u"! as $\mu$.
Exponents such as \lstinline!"m2"! may be presented as m\textsuperscript{2}.
Degrees may be presented as $^{\circ}$, both for \lstinline!"deg"! on its own (for angles) and for temperatures -- e.g., \lstinline!"degC"! can be presented as $^{\circ}$C.
```